src/core/dfn.js

```diff
@@ -238,3 +243,32 @@ function addContractDefaults() {
     dfn.dataset.export = "";
   }
 }
+
+// - Sets data-defines on well-known definition content patterns
+function addDefinitionPointers() {
+  // A dl with class hasdefinitions marks all next siblings of dt with the class
```

Comment:
I don't dislike this (`dl. definitions`), but I feel this is overstepping based on what we have previously discussed. I still think we should start with an extremely simple / basic system and build it up very incrementally (and eventually get to this, but I don't think we are there yet).

Reply:
a `<dl>` containing definitions is exactly what the [VC spec (our initial customer) is using](https://w3c.github.io/vc-use-cases/#terminology) AFAICT; if we're not going to support that convention, I'm not sure we need a respec patch at all at the moment (since reffy recognizes `data-defines` without any additional help).

Reply:
Right, and that might be ok. But let's settle on the convention for:

* container (p, span) has .definition
* and multiple: dl.definitions

And explicitly marked as such.